docs/static/spec/openapi/starter.yaml

```diff
@@ -0,0 +1,252 @@
+
+openapi: 3.1.0
+info:
+  title: Logstash APIs
+  description: |
+    When you run Logstash, it automatically captures runtime metrics that you can use to monitor the health and performance of your Logstash deployment.
+    The metrics collected by Logstash include:
+    
+    - Logstash node info, like pipeline settings, OS info, and JVM info.
+    - Plugin info, including a list of installed plugins.
+    - Node stats, like JVM stats, process stats, event-related stats, and pipeline runtime stats.
+    - Hot threads.
+    - Health report. 
+
+    The APIs that retrieve these metrics are available by default without requiring any extra configuration.
+
+    ## Documentation source and versions
+
+    This documentation is derived from the `main` branch of the [logstash](https://github.com/elastic/logstash) repository.
+    It is provided under license [Attribution-NonCommercial-NoDerivatives 4.0 International](https://creativecommons.org/licenses/by-nc-nd/4.0/).
+  version: '1.0'
+  x-doc-license:
+    name: Attribution-NonCommercial-NoDerivatives 4.0 International
+    url: https://creativecommons.org/licenses/by-nc-nd/4.0/
+  x-feedbackLink:
+    label: Feedback
+    url: https://github.com/elastic/docs-content/issues/new?assignees=&labels=feedback%2Ccommunity&projects=&template=api-feedback.yaml&title=%5BFeedback%5D%3A+
+servers:
+  - url: /
+security:
+  - apiKeyAuth: []
+paths: 
+  /_node/<types>:
+    get:
+      summary: Get node info
+      description: >
+        Get information about Logstash nodes, where `<types>` (optional) specifies the types of node info you want returned.
+        You can limit the info that is returned by combining any of these types in a comma-separated list:
+
+        - `pipelines`. 
+        - `os`. 
+        - `jvm`. 
```

Comment:
What's the trick to getting items on their own lines? (I tried `\n`, but they rendered literally.)

Reply:
Node Info section has subsections for `pipelines`, `os`, and `jvm`, each with their own examples.   
ToDo: Format nested content